sample_app/config/initializers/aws.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Aws.config.update(
+  endpoint: ENV['DYNAMO_ENDPOINT'],
```

Comment:
I believe this changes the endpoint for every service client, not just dynamodb. Was this specifically for aws-record or also aws-sessionstore-dynamodb? Both of these tools have ways to configure the dynamo endpoint. I would prefer that, because global config is icky for reasons like this. If it's infeasible, I believe also you can stick this under the dynamodb key like so:

```
Aws.config.update(dynamodb: { endpoint: ... })
```